cupy/_environment.py

```diff
@@ -2,35 +2,71 @@
 This file must not depend on any other CuPy modules.
 """
 
+import distutils
```

Comment:
I remember there's a PR fixing the import speed by removing `distutils` (or some other libraries), but I can't remember nor find it. Import time is also a concern in my PR.

Reply:
Maybe this one? https://github.com/cupy/cupy/pull/2759

Reply:
That's it! Thanks, Emilio. I wonder if I should just do our own string splitting here and avoid importing distutils?

Reply:
If it's easier it should be preferable.

Reply:
I think `split_quoted()` is an overkill here. It doesn't do input verification for us, and we may as well simply call `split()` to get a list of strings. With a valid input, it'd just do the work; with an invalid input, the command from either `split_quoted()` or `split()` would fail when the subprocess is invoked.